config/runtime.exs

```diff
@@ -61,6 +61,17 @@ if config_env() == :prod do
 
   config :mllparty, boot_connections: boot_connections
 
+  config :sentry,
+    dsn: System.fetch_env!("SENTRY_DSN"),
+    environment_name: :prod,
+    enable_source_code_context: true,
+    root_source_code_path: File.cwd!(),
+    tags: %{
+      env: "production"
```

Comment:
We'll want to take this value as an ENV value so that we can distinguish integration from prod in sentry.